nirc_ehr/resources/etls/historicalOther.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<etl xmlns="http://labkey.org/etl/xml">
+    <name>HistoricalOther</name>
+    <description>Historical data</description>
+    <transforms>
+        <transform id="step1" type="org.labkey.di.pipeline.TransformTask">
+            <description>Copy to historicalOther target</description>
+            <source schemaName="dbo" queryName="q_historicalOther"/>
+            <destination schemaName="study" queryName="historicalOther"
+                         bulkLoad="true" batchSize="5000" targetOption="merge">
```

Comment:
Same as the other ETL, I would recommend targetOption="truncate"

Reply:
Ok, sounds good